### PR TITLE
MONGOID-5180 add release notes for pick

### PR DESCRIPTION
**Pull request by Neilshweky**

### CODE REVIEW

**Review by p-mongo — APPROVED** [review]
I would add whether the sorting is applied.

**Comment by Neilshweky** [comment]
@p-mongo is this alright?

File: docs/release-notes/mongoid-8.0.txt [hunk 1]
```diff
@@ -658,6 +658,43 @@ The ``tally`` method accepts the dot notation and field aliases. It also
...
+  # => 21
+
+This method does not apply a sort to the documents, so it will not necessarily
+return the values from the first document.
+
+The ``pick`` method accepts the dot notation and field aliases. It also
+allows for picking localized fields.
+
+
+
```
Comment (Neilshweky) [inline docs/release-notes/mongoid-8.0.txt:697]:
```suggestion
```

**Review by p-mongo — APPROVED** [review]
:+1:

**Pull request merged**